scalableminds/webknossos
# Put dataset permissions in sharing tab

This PR moves a dataset's permission settings to the Sharing & Permissions tab.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- go to the editing view of a dataset
- there should be a "Sharing & Permissions" tab 
- there should be the input field for teams that are allowed to access the dataset
- in the "Metadata" tab should be no such input field

### Issues:
- fixes #4683

------
